src/Sound.js

```diff
@@ -174,10 +174,12 @@ export default function Sound(props) {
           {/* Description */}
           <div className="pb-16">
             <h1 className="text-left text-3xl pt-24 py-4">Description</h1>
-            <p
-              className="bg-secondary-static text-primary-static border border-primary-static text-lg p-10"
-              dangerouslySetInnerHTML={sanitizedHTMLFromFreesound}
-            />
+            <div className="Description">
+              <p
+                className="bg-secondary-static text-primary-static overflow-scroll border border-primary-static text-lg p-10"
```

Comment:
Move all these classes to parent to fix this

Reply:
@amilajack you said that the max height should be double the current height, which is set to 300px, so I am going to push another commit where the max height is set to 600px. Hope this is what you're looking for.

Reply:
Fixed